python/repositories.bzl

```diff
@@ -53,6 +53,26 @@ def py_repositories():
         ],
     )
 
+    # below are dependencies if you want to use py_proto_library
```

Comment:
this is definitely breaking users when done this way, since it changes the versions of these tools depending what order their WORKSPACE calls various `*_repositories` functions. Like imagine if they install python before Go. I think we need a separate `py_proto_repositories` function in this file instead.

Reply:
I've reverted the changes and I am limiting the scope of the PR to only add an example.